homeassistant/components/awair/const.py

```diff
@@ -39,109 +19,4 @@
 
 DOMAIN = "awair"
 
-DUST_ALIASES = [API_PM25, API_PM10]
-
 LOGGER = logging.getLogger(__package__)
-
-UPDATE_INTERVAL_CLOUD = timedelta(minutes=5)
-UPDATE_INTERVAL_LOCAL = timedelta(seconds=30)
-
-
-@dataclass
-class AwairRequiredKeysMixin:
-    """Mixin for required keys."""
-
-    unique_id_tag: str
-
-
-@dataclass
-class AwairSensorEntityDescription(SensorEntityDescription, AwairRequiredKeysMixin):
-    """Describes Awair sensor entity."""
-
-
-SENSOR_TYPE_SCORE = AwairSensorEntityDescription(
-    key=API_SCORE,
-    icon="mdi:blur",
-    native_unit_of_measurement=PERCENTAGE,
-    name="Score",
-    unique_id_tag="score",  # matches legacy format
-    state_class=SensorStateClass.MEASUREMENT,
-)
-
-SENSOR_TYPES: tuple[AwairSensorEntityDescription, ...] = (
```

Comment:
These are good standardizations though, as these are actually preferred to move to the matching platform in these cases 👍 

../Frenck